apis/blockdevice/v1alpha1/crds.go

```diff
@@ -26,7 +26,7 @@ const (
 	// BlockDeviceResourceShort is the short name used for block device CRD
 	BlockDeviceResourceShort = "bd"
 	// BlockDeviceResourceName is the name of the block device resource
-	BlockDeviceResourceName = BlockDeviceResourcePlural + "." + GroupName
+	BlockDeviceResourceName = BlockDeviceResourcePlural + "." + GroupVersion.Group
 
 	// BlockDeviceClaimResourceKind is the kind of block device claim CRD
 	BlockDeviceClaimResourceKind = "BlockDeviceClaim"
```

Comment:
Why do we need this variables in `blockdevice/crds`. It should be in `blockdeviceclaims/crds` right?